Frameworks/CoreText/CTFont.mm

```diff
@@ -508,19 +601,19 @@ CGFloat CTFontGetUnderlineThickness(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CGFloat CTFontGetSlantAngle(CTFontRef font) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return _DWriteFontGetSlantDegrees(font->_dwriteFontFace);
```

Comment:
we discussed this last time - since this is a C-style function, it probably makes more sense for it to throw than for it to handle null and return a default.

---
In reply to: [80733021](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80733021) [](ancestors = 80733021)